### PR TITLE
Added options to lobby table and game name in-game

### LINKED ISSUE #2733: Title / Description rework
Hi world,
Atm the title text is limited around 44 characters.
This is pretty few it's hard to note which convention you are using if it's not a well known one.

And we got no description of the table. We don't even see the settings in the main page.
I agree that some of the settings are not useful (like it seem bottom deck blind play), but some others look really important (like detrimental characters).

### PR DESCRIPTION
**Pull request by Krycke**
Closes #2733

### CODE REVIEW

**Comment by Krycke** [comment]
options

**Comment by Zamiell** [comment]
neat

**Comment by Zamiell** [comment]
i think the original reason that we didn't do this is that there wasn't enough room.

can you should a screenshot of what it looks like on a small monitor or like a macbook air screen?

**Comment by Krycke** [comment]
options_smaller
options_small
options_smallest

**Comment by Krycke** [comment]
But I guess that mostly people don't play with many options on, which means it will only take up the same space as "Timed" did before.

**Comment by Zamiell** [comment]
cool, looks fine i guess

File: server/src/session_notify.go [hunk 1]
```diff
@@ -102,6 +103,7 @@ func makeTableMessage(s *Session, t *Table) *TableMessage {
 		Owned:             s.UserID == t.OwnerID,
 		Running:           t.Running,
 		Variant:           t.Options.VariantName,
+		Options:           *t.Options,
```
[the remark this answers is not in the document]
Comment (Zamiell) [reply server/src/session_notify.go:106]: i think its idiomatic in golang to pass the slice instead of the slice pointer

**Comment by Krycke** [comment]
Just a miss from my part. The asterixis should be used when declaring the struct and not when values are being assigned. Then at least it's the same as it's done everywhere else.

**Comment by Zamiell** [comment]
whats the infinity / unlimited thing?

**Comment by Krycke** [comment]
It felt strange to have the option empty, and it also felt strange to write
text, so the infinity is unlimited time (not timed).

Den ons 18 jan. 2023 15:46James ***@***.***> skrev:

**Comment by Zamiell** [comment]
i dont really like it, an infinity symbol implies some option

**Comment by Zamiell** [comment]
if the cell looks weird being empty, can we put a raw hyphen

**Comment by Krycke** [comment]
It doesn't really look bad, but right now we are pointing out that the games are not timed. If we don't need to do that, then i can just remove it.

**Comment by Zamiell** [comment]
i think it makes more sense to have the cell be empty now.

previously it made sense to have "not timed" because the cell was only representing one thing.

**Comment by Krycke** [comment]
options_empty

**Comment by Zamiell** [comment]
do you think a hyphen would look better?

**Comment by Krycke** [comment]
Nah, I think it's ok.

**Comment by Krycke** [comment]
options_hyphen

There will just be a clutter of hyphens

**Comment by Zamiell** [comment]
well it looks like you have an extra/bugged ` character there

**Comment by Krycke** [comment]
The code with the hyphen is not working anyway, just a mockup. But shouldn't be that hard to modify into something working. I think an empty cell looks better. And then the committed code should work.

**Comment by Zamiell** [comment]
cool thanks

**Pull request merged**